python/lsst/afw/detection/utils.py

```diff
@@ -19,11 +19,30 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+__all__ = ["writeFootprintAsDefects"]
+
 from . import footprintToBBoxList
 
 
 def writeFootprintAsDefects(fd, foot):
-    """Write foot as a set of Defects to fd"""
+    """
+    Write foot as a set of Defects to fd
+
+    Given a detection footprint, convert it to a BBoxList and write the output to the file object fd.
+    Does not return anything.
+
+    Parameters
+    ----------
+    fd : `typing.TextIO`
+    foot : `lsst.afw.detection.Footprint`
+
+    Returns
+    -------
+
```

Comment:
Normally when we don't have a return value, we leave out the section entirely (I suspect it doesn't format correctly in the built docs, though I can't check). There's also no need to explicitly say "does not return anything."

Reply:
Will adjust accordingly.